.github/workflows/automerge.yml

```diff
@@ -0,0 +1,28 @@
+name: Automerge PR
+on:
+  pull_request:
+    types:
```

Comment:
Do we need all these types?

Reply:
Nevermind, you removed a few. This should be fine then

Reply:
Please ignore - testing that replying does not trigger Automerge

Reply:
Yay it worked: https://github.com/sequelize/sequelize/actions/runs/7944286727